client/configuration/webpack/webpack.config.shared.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
     path: path.resolve("build"),
     publicPath,
   },
+  resolve: {
+    extensions: [".ts", ".tsx", "..."],
```

Comment:
for my own education, what's the `"..."` for? Thank you!

Reply:
`"..."` includes the default webpack values for the `resolve.extensions` array ([docs](https://webpack.js.org/configuration/resolve/#resolveextensions)).

The `resolve.extensions` config option wasn't specified before I added the `.ts` and `.tsx` values; my intention here was to ensure any default values previously being used (e.g. `.js`, `.jsx` or `.css`) would still be supported. Let me know if you would you prefer an explicit list here!

Reply:
Ooh didn't know that! So convenient 😆 Thanks for sharing the knowledge 💡 `"..."` sounds great!